New_PlatformGame/Assets/1. Script/MonsterCtrl.cs

```diff
@@ -8,7 +8,7 @@
 public class MonsterCtrl : MonoBehaviour
 {
     private bool _canMove = true;
-    private float _mobSpeed = 1.5f; //몬스터 속도
+    public float _mobSpeed = 1.5f; //몬스터 속도
```

Comment:
```suggestion
    public float _mobSpeed = 1.5f; //몬스터 속도
```
public으로 만들었으면 라이더에서 지원하는 rename 기능으로 한번에 변수 이름을 싹 바꾸도록 하자.